src/libraries/System.Private.CoreLib/src/System/Numerics/Vector.cs

```diff
@@ -90,102 +90,102 @@ public static Vector<TTo> As<TFrom, TTo>(this Vector<TFrom> vector)
 #endif
         }
 
-        /// <summary>Reinterprets a <see cref="Vector{T}" /> as a new <see cref="Vector{Byte}" />.</summary>
+        /// <summary>Reinterprets a <see cref="Vector{T}" /> as a new <see langword="Vector{Byte}" />.</summary>
```

Comment:
```suggestion
        /// <summary>Reinterprets a <see cref="Vector{T}" /> as a new <see langword="Vector<Byte>" />.</summary>
```

Would angle brackets be better?

Reply:
Or maybe angle brackets aren't allowed.

Reply:
I think they are allowed but they need to be escaped. Take a look at the API DocIDs.

Reply:
Fixed.